### ISSUE TITLE
Cleanup calls the NotifyOnTick and NotifyOnEpoch

### ISSUE THREAD

**Issue by jeremyletang** [vegaprotocol/vega#4625]
As of now we register NotifyOnTick and NotifyOnEpoch in many places:
- sometimes twice 
- sometimes now some needs to be registered in a specific order
- sometimes from the cmd/vega/node package
- sometimes from types constructors

Also some type are tacking a initial time when being constructred, which is always wrong as types are constructed before the blockchain is started, and no time is available before Genesis, so these should be constructed without the time, and register a callback anyway.

At least all top level types should register NotifyOnTick on a single place, so we could track of them all, and ensure they are registered only once, and with an order which is correct.

**Comment by ze97286** [comment]
this was already covered by #5475
